constants/chatbotCommands.py

```diff
@@ -129,6 +129,22 @@ async def _help(fro: str, chan: str, message: list[str]) -> str:
     return "\n".join(l)
 
 
+@command(
+    trigger="!addbn",
+    syntax="<name>",
+    privs=privileges.ADMIN_MANAGE_PRIVILEGES,
+)
+async def addbn(fro: str, chan: str, message: list[str]) -> str:
+    """Add BN priveleges to a user"""
+    userid = await user_utils.get_id_from_username(fro)
```

Comment:
`fro` is the username of the person who invoked the command -- so at the moment the logic will currently add bn privileges to only them (and not the person in the !addbn) command

We'll need to get the target username from the `message` list.

This works the following way:

If `Mahmood` sends the following message in the `#osu` chat: `!addbn cmyui`

The following variables will be present:
```
fro: "Mahmood"
chan: "#osu"
message: ["cmyui"]
```

So, to get the username from the message, we need to get the first index -- i.e. `target_username = message[0]`.